lib/tests/test_workspace.rs

```diff
@@ -26,10 +26,10 @@ fn test_load_bad_path() {
     let workspace_root = temp_dir.path().to_owned();
     // We haven't created a repo in the workspace_root, so it should fail to load.
     let result = Workspace::load(&settings, &workspace_root, &BackendFactories::default());
-    assert_eq!(
+    assert!(matches!(
```

Comment:
We already have a dependency on `assert_matches`, so use that here if possible (gives better error messages?)

Reply:
Ah, nice, didn't see that was available!